html/modules/custom/docstore/src/SearchableResourceTrait.php

```diff
@@ -233,6 +233,23 @@ public function searchResources(Request $request, $entity_type_id, ?ConfigEntity
         $data['revisions'] = $this->getResourceEntityRevisionList($entity_type_id, $id);
       }
     }
+    else {
```

Comment:
What about metadata when requesting files only?

Reply:
True, needs to be added